drop-storage/.env

```diff
@@ -1 +1 @@
-DATABASE_URL="sqlite://testdb.sqlite3"
+DATABASE_URL="sqlite://drop-storage/testdb.sqlite3"
```

Comment:
I would change the file name to `.env.example` or something like that so that it won't create sqlite files somewhere accidentally

Reply:
This file is used to tell sqlx where the database is so it could verify the `sqlx::query!` calls, without it the build would fail, unless we define the variable in the build command, and it also messes up linting in IDE's because of the same problem

Reply:
A better solution is to remove `.env` and `testdb.sqlite` and use the offline mode of sqlx https://github.com/launchbadge/sqlx/tree/main/sqlx-cli#enable-building-in-offline-mode-with-query

Reply:
changed it to offline mode, one additional idea might be to add the `sqlite` and `env` files to `gitignore` so that locally you could use online mode, since it takes precedence

Reply:
Um, it's not what I meant by offline mode. My point was to not include the `offline` feature but to run `cargo sqlx prepare` then it generates the `<something>.sqlx` file which is referred to be the macros during the compilation. The `DATABASE_URL` still takes precedence but it's no longer needed if you're compiling it on CI or do not care about SQL development at this moment.
Regarding the `.env` it's a good idea to add it to `.gitignore` :+1:

Reply:
I've updated the `.gitignore` file

As for the `offline` feature, this is actually needed to generate and parse the `****.sqlx` file, `sqlx` complained about it during the generation, and it makes sense if you look at the diff in the `Cargo.lock` file, as it pulls in `serde` dependencies

Reply:
Are you sure? This is from sqlx README
![image](https://github.com/NordSecurity/libdrop/assets/41059082/560bb8bf-93ca-4d16-83fe-c9fe1b9a3ecc)

Reply:
This is the current README for the upcoming `0.7.0` version, if we look at the readme from the `0.6` version that we are using
![image](https://github.com/NordSecurity/libdrop/assets/1346026/99afb6aa-5701-4a21-9496-68a47756b0bb)

source: https://github.com/launchbadge/sqlx/blob/17cebde64aa6313d8c3efe2db6a05bdcff48def0/README.md

Reply:
Ok, my bad :facepalm: